theia-full-docker/Dockerfile

```diff
@@ -182,8 +182,8 @@ ARG GITHUB_TOKEN
 RUN yarn --cache-folder ./ycache && rm -rf ./ycache && \
     NODE_OPTIONS="--max_old_space_size=4096" yarn theia build
 EXPOSE 3000
-ENV SHELL /bin/bash\
-    THEIA_DEFAULT_PLUGINS=local-dir:/home/theia/plugins
+ENV SHELL /bin/bash
+ENV THEIA_DEFAULT_PLUGINS local-dir:/home/theia/plugins
```

Comment:
This works much better. 

Consider using the [second form](https://github.com/theia-ide/theia-apps/pull/265#discussion_r354414132) for `ENV` that permits defining multiple key-value pairs and saves us from adding extra docker image layers.

Reply:
Fixed